internal/server/agentProxy.go

```diff
@@ -213,8 +212,8 @@ func (proxy *AgentProxy) defaultHandler(w http.ResponseWriter, r *http.Request)
 	}
 
 	log.Info("Sending response")
-	n, _ := io.Copy(w, res.Body)
-	LogExit(log, r, res.StatusCode, int(n))
+	w.WriteHeader(http.StatusOK)
+	_, _ = io.Copy(w, res.Body)
```

Comment:
Isn't it better to just do the function call instead of getting a blank identifier for all the call results? 
```suggestion
	io.Copy(w, res.Body)
```

Reply:
Indeed! i did it initially, however to avoid alerts from golangci-lint, i kept it blank. The another packages follow the same way.

Reply:
@vinijabes its a good pratice to explicity tell when you are ignoring an error. Because sometimes that can be a bug implemented in the code.

Reply:
Thanks for the explain @racerxdl!